gst/nnstreamer/tensor_common.h

```diff
@@ -111,7 +111,8 @@ typedef struct
  * For the concrete definition of headers, please look at the wiki page of nnstreamer:
  * https://github.com/nnsuite/nnstreamer/wiki/Design-External-Save-Format-for-other-tensor-and-other-tensors-Stream-for-TypeFind
  */
-extern void gst_tensors_typefind_function (GstTypeFind * tf, gpointer pdata);
+extern void
+gst_tensors_typefind_function (GstTypeFind * tf, gpointer pdata);
 
```

Comment:
This should go other commit.

Reply:
I added new PR about type-find function.